flycheck-pos-tip.el

```diff
@@ -66,17 +68,23 @@ The function should be a no-op in this case."
   :package-version '(flycheck-pos-tip . "0.2"))
 
 (defcustom flycheck-pos-tip-timeout 5
-  "Time in seconds to hide the tooltip after."
+  "Time in seconds to hide the tooltip after.
+
+This setting currently affects only the default
+`flycheck-pos-tip-show-function'."
   :group 'flycheck-pos-tip
   :type 'number
   :package-version '(flycheck-pos-tip . "0.2"))
 
 (defun flycheck-pos-tip-show (messages)
-  "Show a pos-tip popup with MESSAGES.
+  "Show MESSAGES, using a pos-tip popup on GUI frames.
 
+On TTY frames, fall back to `flycheck-display-error-messages'.
 Uses `pos-tip-show' under the hood."
-  (pos-tip-show (mapconcat #'identity messages "\n\n") nil nil nil
-                flycheck-pos-tip-timeout))
+  (if (display-graphic-p)
+      (pos-tip-show (mapconcat #'identity messages "\n\n")
+                    nil nil nil flycheck-pos-tip-timeout)
+    (flycheck-display-error-messages messages)))
```

Comment:
I'm not sure whether that actually works.  As far as I can tell, `flycheck-display-error-messages` wants a list of error objects as argument.  We might need to change `flycheck-pos-tip-function` so that it takes a list of errors instead strings…